client/extensions/woocommerce/state/sites/shipping-classes/test/actions.js

```diff
@@ -0,0 +1,76 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	WOOCOMMERCE_SHIPPING_CLASSES_REQUEST,
+	WOOCOMMERCE_SHIPPING_CLASSES_REQUEST_SUCCESS,
+} from './../../../action-types';
+import {
+	fetchShippingClassesSuccess,
+	fetchShippingClassesFailure,
+	fetchShippingClasses,
+} from '../actions';
+import initialShippingClasses from './data/initial-state';
+
+const siteId = 123;
+
+const dispatchFn = action => action;
```

Comment:
If you actually need the return value (which I don't see being used) you can swap this for Lodash's `identity`, otherwise `noop` could work.

Reply:
The return value is beeing used in some cases, ex. `fetchShippingClasses`:

```js
return dispatch( {
	type: WOOCOMMERCE_SHIPPING_CLASSES_REQUEST,
	siteId,
} );
```

Functionally we wouldn't need the return, but I added it for cleaner-looking tests.